scripts/release-bit-verification.sh

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/sh
+
+############################################################################
+#  PREREQUISITES FOR THIS SCRIPT
+# 1. Redistributable-binary(.rpm) should be passed as the first argument
+# 2. Login to the cluster should be done prior to running this script
+# 3. The cluster should be in a state where it can be used for testing
+#
+# USAGE:
+# ./release-bit-verification.sh redistributable-binary
+#
+# Example: ./release-bit-verification.sh ~/Downloads/odo-redistributable-2.4.3-1.el8.x86_64.rpm
+#
+
+shout() {
+  echo "--------------------------------------------------------------------------"
+}
+# Check SHASUM for all the binary files and there should be no difference
+
+# Create a Temp directory 
+WORKING_DIR=`mktemp -d`
+
+# Extract from rpm file 
+rpm2cpio ${1} | cpio -idmvD $WORKING_DIR
+pushd $WORKING_DIR/usr/share/odo-redistributable/
+
+# Check sha256sum for all the files
+while IFS= read -r line; do
+    read -r SHA FILE <<<"$line"
+    read -r SHATOCHECK FILE <<< `sha256sum $FILE`
+    if [[ $SHA == $SHATOCHECK ]]; then
+        # Print if the file is correct
+        printf '%-50s\U0002705\n' $FILE 
+    fi
+done < SHA256_SUM
+
+shout
+
+# Copy binary for testing purpose
+OS=`uname -s`
+ARCH=`uname -m`
+
+if [[ $OS == "Linux" ]]; then
+    if [[ $ARCH == "x86_64" ]]; then
+        cp ./odo-linux-amd64 $GOBIN/odo
```

Comment:
GOBIN might not always be set, and this command would fail.
Who is going to use this script?

Reply:
Mostly QE will use this script.

Reply:
>Mostly QE will use this script.

There will be time when someone else needs to execute this script, and this will definitely make it fail.

Cleaner approach would be to save bin path to variable and use it  later when calling odo commands.
For example like this 
```bash
ODO="./odo-linux-amd64"
```
and then use it `$ODO version`

Or if you want to keep using `odo` in the rest of the script you can also do someting like this
```bash
cp ./odo-linux-amd64 odo
PATH=`pwd`:$PATH
```

Reply:
ah, now I noticed that you are using `make test-integration-devfile` to run tests.
So you will have to go with second apporach and do 
```
export  PATH=`pwd`:$PATH
```

Reply:
Thanks, updated the with changes.